ixdiagnose/config.py

```diff
@@ -25,6 +26,7 @@ def __init__(self):
         self.debug_path: Optional[str] = None
         self.exclude_artifacts: List[str] = []
         self.exclude_plugins: List[str] = []
+        self.extra: Dict[str, str] = {}
```

Comment:
Should we rename this to `extra_artifacts` perhaps or something so it reflects what extra it is going to actually do ?